lib/src/dart3_suggestors/null_safety_prep/required_flux_props.dart

```diff
@@ -0,0 +1,247 @@
+// Copyright 2023 Workiva Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:analyzer/dart/analysis/results.dart';
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/dart/element/type.dart';
+import 'package:collection/collection.dart';
+import 'package:over_react_codemod/src/util.dart';
+import 'package:over_react_codemod/src/util/class_suggestor.dart';
+import 'package:over_react_codemod/src/util/offset_util.dart';
+
+/// Suggestor that adds required `store` and/or `actions` prop(s) to the
+/// call-site of `FluxUiComponent` instances that omit them since version
+/// 5.0.0 of over_react makes flux `store`/`actions` props required.
+///
+/// In the case of a component that is rendered in a scope where a store/actions
+/// instance is available, but simply not passed along to the component, those
+/// instance(s) will be used as the value for `props.store`/`props.actions`,
+/// even though the component itself may not make use of them internally.
+///
+/// In the case of a component that is rendered in a scope where a store/actions
+/// instance is not available, `null` will be used as the value for the prop(s).
+class RequiredFluxProps extends RecursiveAstVisitor with ClassSuggestor {
+  ResolvedUnitResult? _result;
+
+  static const fluxPropsMixinName = 'FluxUiPropsMixin';
+
+  @override
+  visitCascadeExpression(CascadeExpression node) {
+    final cascadeWriteEl = node.staticType?.element;
+    if (cascadeWriteEl is! ClassElement) return;
+    final isReturnedAsDefaultProps = node.ancestors
+            .whereType<MethodDeclaration>()
+            .firstOrNull
+            ?.name
+            .lexeme
+            .contains(RegExp(r'getDefaultProps|defaultProps')) ??
+        false;
+    if (isReturnedAsDefaultProps) return;
+
+    final maybeFluxUiPropsMixin = cascadeWriteEl.mixins
+        .singleWhereOrNull((e) => e.element.name == fluxPropsMixinName);
+    if (maybeFluxUiPropsMixin == null) return;
```

Comment:
I noticed a lot of cases of `PanelTitle` and `PanelTitleV2`, which happen to mix in FluxUiPropsMixin, but doesn't actually use those props. 

Adding/props/store to these is pretty noisy, and as part of FED-2015 I think we'll suppress required prop errors for them, so I feel like it would be worth also making an exception for them. 

I think you should be able to do that with this:
```dart
    const typesToIgnore = {'_PanelTitleProps', 'PanelTitleProps', 'PanelTitleV2Props'};
    if (typesToIgnore.contains(cascadeWriteEl.name)) {
      return;
    }
```

Reply:
Fixed / added tests in https://github.com/Workiva/over_react_codemod/pull/268/commits/0f3a754e5c6a5c4913b6c000cc8910a2e6c21c1a